supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -149,6 +151,13 @@ object PipelinesApiConfigurationAttributes {
       case other => throw new IllegalArgumentException(s"Unrecognized caching duplication strategy: $other. Supported strategies are copy and reference. See reference.conf for more details.")
     } }
     val requestWorkers: ErrorOr[Int Refined Positive] = validatePositiveInt(backendConfig.as[Option[Int]]("request-workers").getOrElse(3), "request-workers")
+
+    val pipelineTimeout: Option[FiniteDuration] = backendConfig.as[Option[FiniteDuration]]("pipeline-timeout") match {
+      case Some(duration) if duration.isFinite() => Option(duration)
+      // Defaults to Google's defined default (7d)
+      case None => Option(7.days)
+    }
```

Comment:
Is there a reason the `FiniteDuration` has the `if duration.isFinite()` guard?

It looks like this is always being assigned, in which case this might be simplified

```
    val pipelineTimeout: FiniteDuration = backendConfig.getOrElse("pipeline-timeout", 7.days)
```
with the cascading changes to de-`Option` the `pipelineTimeout` elsewhere.

Reply:
I think this makes more sense, and it also makes it simpler to set the timeout in the GenomicsFactory! Changed.